test/fixtures/state-saved-v1-5.js

```diff
@@ -214,6 +215,9 @@ export const savedStateV1TripGeoJson = {
           brush: {
             size: 0.5,
             enabled: false
+          },
+          geocoder: {
```

Comment:
the saved state should not be changed, the merged config should have geocoder set to `{enablced: false}`